### DISCUSSION TITLE
v0.26.0 / Prog_Steuerung.sqlite / Docker-Version

### DISCUSSION THREAD

**Discussion by jaydee73** [wiggal/GEN24_Ladesteuerung#117] [Q&A]
Hi wiggal,

kurze Frage zur Docker-Version der 0.26.0. Im Changelog steht ja, dass die Prog_Steuerung.sqlite "aus den Quellen entfernt" wurde. Bedeutet das, dass man bei der Docker-Version diese Datei auch nicht mehr in den Container mappen muss? Enthält die sqlite denn keine sich ändernden Daten? Weil mit jedem Docker-Pull würde sie damit ja quasi wieder gelöscht/neu generiert, wenn man sie nicht explizit reinmapped.

Oder habe ich da einen Verständnisfehler?

Gruß,
JD

**Comment by wiggal** [comment]
Die Datei enthält schon sich ändernde Daten, das war ja das Problem. Wenn man das Update per git pull machte wurde die eigene Prog_Steuerung.sqlite mit der aus den Quellen überschrieben. Ich hoffe es ist im Docker kein Problem die Datei zu mappen, wenn sie nicht im Image ist.

**Reply by jaydee73** [reply]
Ok, also lasse ich sie weiterhin gemapped. Dass sie nicht im Image ist, sollte ja egal sein. Und solange sich der Pfad nicht geändert hat, wo dein Script die Datei erwartet, sollte er die bestehende sqlite ja auch einfach weiternutzen, ohne dass eine neue erzeugt wird (und damit die Bestandsdaten ignoriert/überschrieben werden).

**Comment by jaydee73** [comment]
@wiggal : Ich wollte das Thema nochmal aufgreifen. Habe gerade gesehen, dass meine (via Docker gemappte) prog_steuerung.sqlite seit dem 18.11.25 anscheinend durch den Container nicht mehr angefasst wurde (also nicht mehr geändert wurde). Daher nochmal die aktualisierte Frage: Wird die gemappte Datei noch weiter gebraucht? Oder kann ich auf das Mapping mittlerweile verzichten?

**Reply by wiggal** [reply]
Wie damals schon geschrieben wird die Prog_Steuerung.sqlite schon benutzt und auch beschrieben. Die zu mappenden Dateien kannst du unter DOCKER/docker-compose.yml nachsehen.

**Reply by wiggal** [reply]
Schau das Mapping nochmal genau an

**Reply by jaydee73** [reply]
hmmm...Habe ich gemacht. Ich bin der Meinung, das Mapping stimmt:

<img width="1205" height="171" alt="Screenshot 2026-06-02 at 21-07-23" src="https://github.com/user-attachments/assets/bd5ef860-8b27-4ef0-b119-ae8d6b545a4f" />

Das ist der Pfad aus de yml-Datei. Ich schau es mir nochmal an...

**Reply by wiggal** [reply]
sobald du im LadeStrg-Tab auf Ladersteuerung speichern klickst, schreibt er in die Prog_Steuerung.sqlite und das Datz ändert sich

**Reply by jaydee73** [reply]
ah ok, dann war das das Problem, bzw. mein Denkfehler. Ich nutze diesen Button idR nicht. Daher wurde dann meine progsteuerung-Datei auch nie/selten beschrieben.

Danke!
